frontend/src/app/search/SearchForm.tsx

```diff
@@ -87,29 +87,26 @@ export function SearchForm({
               initialQueryParams={sortbyQueryParams}
             />
             <div className="usa-prose">
-              {searchResults?.data.length >= 1 ? (
-                <SearchPagination
-                  totalPages={searchResults?.pagination_info?.total_pages}
-                  page={page}
-                  handlePageChange={handlePageChange}
-                  showHiddenInput={true}
-                  paginationRef={topPaginationRef}
-                />
-              ) : null}
```

Comment:
The `SearchForm` file is all jsx right now except for these conditionals, pull this check into the `SearchPagination` component